groups/sig-release/groups.yaml

```
@@ -398,6 +398,10 @@ groups:
       - michael.levan@clouddev.engineering # 1.28 Docs Shadow
       - taniaduggal60@gmail.com # 1.28 Docs Shadow
       - vibhorchinda@gmail.com # 1.28 Docs Shadow
+      - a.zelyk@reply.de # 1.28 Bug Triage Shadow
```

let's align this with https://github.com/kubernetes/k8s.io/pull/5272
```suggestion
      - zelikangelina@gmail.com # 1.28 Bug Triage Shadow
```